### ISSUE TITLE
Class prediction 

### ISSUE THREAD

**Issue by abhigoku10** [ghimiredhikura/Complex-YOLO#2]
@ghimiredhikura  Hi i trained the model using you training code , i have few queries below 

1. When i tested the trained model withe validation set the output results were bad 
2. The output tensors 7:14 had negative values
3. Which are the class labels used for training and testing 

Thanks in advance

**Comment by abhigoku10** [comment]
solved

**Comment by ghimiredhikura** [comment]
You mean, it is solved in last commit?

**Comment by abhigoku10** [comment]
@ghimiredhikura  nope i debugged it and solved it

**Comment by ghimiredhikura** [comment]
@abhigoku10, can you post what was the problem and how you solved it.

**Comment by asimonov-hitachi** [comment]
yes, after my training the predicted boxes are good, but classes are pretty much random.
is it training or inferrence interpretation problem?

**Comment by asimonov-hitachi** [comment]
is the fix to use sigmoid here?

https://github.com/abhigoku10/Complex-YOLO/blob/master/eval.py#L47

**Comment by abhigoku10** [comment]
> is the fix to use sigmoid here?
> 
> https://github.com/abhigoku10/Complex-YOLO/blob/master/eval.py#L47

This the fix during evaluation/ testing . For training you have include the same in  https://github.com/abhigoku10/Complex-YOLO/blob/master/region_loss.py #L205

**Comment by asimonov-hitachi** [comment]
ah, yes. thanks @abhigoku10 !
